Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/client/cache/fifo_test.go

```diff
@@ -160,3 +160,27 @@ func TestFIFO_detectLineJumpers(t *testing.T) {
 		t.Fatalf("expected %d, got %d", e, a)
 	}
 }
+
```

Comment:
Should there be a test like TestFIFO_basic that randomly does an `AddIfNotPresent` instead of just `Add` ?